### DISCUSSION TITLE
How can I remove the title of folder?

### DISCUSSION THREAD

**Discussion by Ali-Hussein-dev** [fuma-nama/fumadocs#903] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/c7464cd0-42c9-40d8-87c0-fbd3e240ed13)

I am using 
`"fumadocs-core": "13.4.9"
    "fumadocs-mdx": "10.0.2"
    "fumadocs-ui": "13.4.9"`

**Comment by fuma-nama** [comment, marked as answer]
change the `breadcrumbs` settings from `DocsPage`

**Reply by Ali-Hussein-dev** [reply]
Thanks
